tendrl/gluster_integration/sds_sync/client_connections.py

```diff
@@ -20,8 +20,9 @@ def sync_volume_connections(volumes):
                         brick_name.split(":")[0],
                         brick_name.split(":_")[-1]
                     ).load()
-                    vol_connections += 0 if fetched_brick.client_count == '' \
-                        else int(fetched_brick.client_count)
+                    if fetched_brick.client_count:
```

Comment:
Please check if "subvol" is none on L16 and also check if "fetched_brick" is None on L23